backend-rust/src/indexer.rs

```diff
@@ -2907,15 +2961,18 @@ async fn process_cis2_event(
 
 struct PreparedScheduledReleases {
     account_address: Arc<String>,
-    release_times:   Vec<DateTime<Utc>>,
-    amounts:         Vec<i64>,
-    update_balance:  PreparedUpdateAccountBalance,
+    release_times: Vec<DateTime<Utc>>,
+    amounts: Vec<i64>,
+    target_account_balance_update: PreparedUpdateAccountBalance,
+    source_account_balance_update: PreparedUpdateAccountBalance,
```

Comment:
It might be we only have to make this a PreparedAccountStatement, I just do not understand why we are not updating the account as well.

Reply:
I think this should update balance of both sender and receiver immediately, so it look correct now :+1: